puble.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>publishing_m</title>
+    <link rel="stylesheet" href="puble.css" />
+  </head>
+  <body>
+    <body class="container">
+      <header class="header">
```

Comment:
header 에 굳이 클래스로 header 를 추가할 필요는 없어 보입니다!
```suggestion
      <header>
```